generated/nidcpower/nidcpower/lcr_load_compensation_spot.py

```diff
@@ -0,0 +1,144 @@
+import ctypes
+import numbers
+import platform
+
+import nidcpower._visatype
+import nidcpower.enums as enums
+
+
+# This class is an internal implementation detail
+# ctypes definition
+# Name must match exactly what the name of the structure type is named in the C API.
+class struct_NILCRLoadCompensationSpot(ctypes.Structure):  # noqa N801
+    if platform.system() == "Windows" and platform.architecture()[0] == "64bit":
+        _pack_ = 8
+    elif platform.system() == "Linux" or platform.system() == "Darwin":
+        _pack_ = 4
+    else:
+        _pack_ = 1
+    _fields_ = [
+        ("frequency", nidcpower._visatype.ViReal64),
+        ("reference_value_type", nidcpower._visatype.ViInt32),
+        ("reference_value_a", nidcpower._visatype.ViReal64),
+        ("reference_value_b", nidcpower._visatype.ViReal64),
+    ]
+
+    def __init__(self, data=None):
+        super(ctypes.Structure, self).__init__()
+        if data is not None:
+            self.frequency = data.frequency
+            self.reference_value_type = enums.LCRReferenceValueType(data.reference_value_type).value
+            if isinstance(data, LCRLoadCompensationSpot):
```

Comment:
waiting on this

Reply:
Constructing from an `LCRLoadCompensationSpot` is the only necessary option as it would be used by the `perform_lcr_load_compensation()` method to be added as part of #1772 (https://github.com/ni/nimi-python/pull/1772/files#diff-31582e14c636470681a52cfa45ebf6f28dbd68afde8827d6058a43022b3231c4R5716). I came up with the other code paths based on https://github.com/ni/nimi-python/blob/master/src/niscope/custom_types/waveform_info.py

I have removed the other unnecessary code paths in my latest commit.